politeiad/api/v1/v1.go

```diff
@@ -357,6 +357,7 @@ type Inventory struct {
 	IncludeFiles bool `json:"includefiles"` // Include files in records
 	// XXX add VettedStart and BranchesStart
 	VettedCount   uint `json:"vettedcount"`   // Last N vetted records
+	VettedPage    uint `json:"vettedpage"`    // Page for vetted records
```

Comment:
Can you make this `VettedStart` per the comment on line 358.  You do this anyway in the code ` 		pageStart := int(vettedCount * vettedPage)`.